tests/lib/rules/comma-style.js

```diff
@@ -233,7 +233,49 @@ ruleTester.run("comma-style", rule, {
                     NewExpression: true
                 }
             }]
+        },
+        "var foo = [\n , \n 1, \n 2 \n];",
+        {
+            code: "const [\n , \n , \n a, \n b, \n] = arr;",
+            output: "const [\n , \n , \n a, \n b, \n] = arr;",
```

Comment:
“output” is only used for tests that fail. Can you please remove these in each passing test?

Reply:
Hi @nzakas, thank you for the review. I changed the test file.